client/src/components/product.js

```diff
@@ -0,0 +1,26 @@
+//Display info for 1 product
+
+import React from 'react';
+import { Component } from 'react';
+
+export default class Product extends Component {
+    render() {
+        const product = this.props.product;
```

Comment:
This is currently breaking your app because `product` is not defined, so when you try to access its category and price property below, this results in a reference error. This is an instance where you would want to connect to your redux store, given that your products were already included in it. Then, via mapStateToProps, you could connect to your products state, and map over them in order to generate this "Card" for every product. Redux aside, you aren't passing a product prop to this component higher up in the component/app tree, so it doesn't have a reference to `this.props.product`.

Reply:
Thank You!!!!!